utils/token_utils.go

```diff
@@ -184,146 +139,6 @@ func (config *TokenConfig) verifyCreateWLCG() error {
 	return nil
 }
 
-func verifyCreateWLCG(claimsMap *map[string]string) error {
-	/*
-		Don't check for the following claims because ALL base tokens have them:
-		- iat
-		- exp
-		- nbf
-		- iss
-		- jti
-	*/
-	if len(*claimsMap) == 0 {
-		return errors.New("To create a valid wlcg, the 'aud' and 'sub' claims must be passed, but none were found.")
-	}
-
-	requiredClaims := []string{"sub", "wlcg.ver", "aud"}
-	for _, reqClaim := range requiredClaims {
-		if val, exists := (*claimsMap)[reqClaim]; !exists {
-			// we can set wlcg.ver because we know what it should be
-			if reqClaim == "wlcg.ver" {
-				(*claimsMap)["wlcg.ver"] = "1.0"
-			} else {
-				// We can't set the rest
-				errMsg := "The claim '" + reqClaim +
-					"' is required for the wlcg profile, but it could not be found."
-				return errors.New(errMsg)
-			}
-		} else {
-			if reqClaim == "wlcg.ver" {
-				verPattern := `^1\.[0-9]+$`
-				re := regexp.MustCompile(verPattern)
-				if !re.MatchString(val) {
-					errMsg := "The provided version '" + val + "' is not valid. It must be of the form '1.x'"
-					return errors.New(errMsg)
-				}
-			}
-		}
-	}
-
-	return nil
-}
-
-func CreateEncodedToken(claimsMap map[string]string, profile TokenProfile, lifetime int) (string, error) {
```

Comment:
Why are all these being deleted?  Were they never used (and we decided they won't be used)?

Reply:
This is a fix-up from previous PR #429 where I replaced the existing claim-map based token creation by the OOP based token creation but forget to remove the code. Nothing would be broken, I've updated code and test to use the new `CreateToken` function instead.